@types/signup.d.ts

```diff
@@ -16,6 +16,6 @@ export type ErrorDisplayProps = {
 }
 
 export type SignupErrors = {
-  userName?: string[]
-  confirmEmail?: string[]
+  userName?: boolean
+  confirmEmail?: boolean
```

Comment:
When we use boolean for a variable it's a good practice to start their name with **is/has**:
- `hasName` | `hasEmail`
- `isUserNameValid` | `isEmailValid`